controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -1227,7 +1227,7 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
 
         const schemaChanges = await getDiffBetweenGraphs(subgraph.schemaSDL, newSchemaSDL);
         if (schemaChanges.kind === 'failure') {
-          logger.debug(`Error finding diff between graphs: ${schemaChanges.error}`);
+          logger.error(`Error finding diff between graphs: ${schemaChanges.error}`);
```

Comment:
This is a user error correct? We should never log users errors as errors because we have infrastructure alerts on it.

Reply:
will logger.warn also trigger a alert?

debug logs are not seen on the log explorer as I think its configured with info

Reply:
Yes, let's do warn.

Reply:
Done, changed to warning